source/connect/tls.txt

```diff
@@ -0,0 +1,340 @@
+cryptographic certificates that prove identity for the application
+itself and other applications with which the application
+interacts. You can configure access to these certificates in your application with
+the following mechanisms:
+
+- The JVM trust store and JVM key store
+- A client-specific trust store and key store
+
+.. note::
+
+   The following sections are based on the documentation for Oracle JDK,
+   so some parts may be inapplicable to your JDK or to the custom TLS/SSL
+   implementation you use.
+
+.. _tls-configure-jvm-truststore:
+
+Configure the JVM Trust Store
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+
+   By default, the JRE includes many commonly used public certificates
+   from signing authorities like `Let's Encrypt
+   <https://letsencrypt.org/>`__. As a result, you can connect to
+   instances of :atlas:`MongoDB Atlas </?jmp=docs_driver_kotlin>` (or any other
+   server whose certificate is signed by an authority in the JRE's default
+   certificate store) with TLS/SSL without configuring the trust store.
+
+The JVM trust store saves certificates that securely identify other
+applications with which your {+language+} application interacts. Using these
+certificates, your application can prove that the connection to another
+application is genuine and secure from tampering by third parties.
+
+If your MongoDB instance uses a certificate that is signed by an
+authority that is not present in the JRE's default certificate store,
+your application must configure two system properties to initiate
+SSL/TLS requests. These properties ensure that your application can
+validate the TLS/SSL certificate presented by a connected MongoDB instance.
+
+- ``javax.net.ssl.trustStore``: the path to a trust store containing the
+  certificate of the signing authority
+
+- ``javax.net.ssl.trustStorePassword``: the password to access the trust
+  store defined in ``javax.net.ssl.trustStore``
+
+You can create a trust store with the `keytool
+<https://docs.oracle.com/javase/8/docs/technotes/tools/unix/keytool.html>`__
+command line tool provided as part of the JDK:
+
+.. code-block:: console
+
+   keytool -importcert -trustcacerts -file <path to certificate authority file>
+            -keystore <path to trust store> -storepass <password>
+
+Configure the JVM Key Store
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+
+   By default, MongoDB instances do not perform client certificate
+   validation. You must configure the key store if you configured your MongoDB
+   instance to validate client certificates.
+
+The JVM key store saves certificates that securely identify your {+language+}
+application to other applications. Using these certificates, other
+applications can prove that the connection to your application is
+genuine and secure from tampering by third parties.
+
+An application that initiates TLS/SSL requests needs to set two JVM system
+properties to ensure that the client presents a TLS/SSL certificate to
+the MongoDB server:
```

Comment:
S: use source constant

Reply:
Q: referring to the "MongoDB server"? The source constant has it formatted as a proper noun with leading capitalization (`MongoDB Server`), is that okay?

Reply:
lgtm